psf_proposal-SCiCookie2.md

```diff
@@ -0,0 +1,144 @@
+  technologies. Via configuration options, each user is able to
+  establish their unique set of dependencies. With the work planned for
+  this grant, the changes will allow an improved user experience while
+  guaranteeing a robust project template options.
+
+  SciCookie Team also plans to apply for the PyOpenSci review process, so this
+  grant will be used to review all the evaluation criteria, present the
+  project there and apply the suggestions from the review process. It is
+  very important for us to be able to apply to PyOpenSci as it is, if
+  accepted, will allow us to raise awareness of our package and thus
+  contribute to the development of the Python and open source community.  
+
+  This cookiecutter allows you to automate the process of
+  creating a Python project using a template that has updated
+  technologies that are considered the most appropriate. The options can
+  be configured according to our needs.
+
+  SciCookie Team looking for Open Science Labs to grow and to be known
+  by the Python community, we have promoted some Python meetings
+  (pycafe) online. In the OSL discord the SciCookie Team maintain a
+  Python group, divided by various topics of interest, and we spread
+  Python through the various projects in incubation that use it.
+
+- Beneficiaries *:
+
+  In general, although we are focusing on the scientific Python
+  community, this cookiecutter template could be used by anyone who
+  wants to create a Python package.
+
+  With this grant, we plan to contract 4 people from our current group
+  of collaborators in OSL. The main two developers are Yurely Camacho
+  and Anavelyz Perez based in Merida-Venezuela, recently graduated in
+  statistics and they are enthusiastic about open science and
+  programming languages like Python. They contributed to the translation
+  of the documentation at Spanish of the library scikit-learn. Each of
+  them will have an hourly intensity of 70 hours during the eight weeks
+  of the project. Additionally, a Project manager and one Senior
+  Developer for Review and quality assurance, these individuals will
+  have an hourly intensity of 10 and 20 hours, respectively.
+
+  This grant will also be accompanied and mentored mainly by Ivan Ogasawara, a
+  member of the Open Science Labs steering council. He has been chair of
+  the SciPy Latinoamerica (SciPyLA) 2016 conference. He had been a
+  maintainer of the SciPyLA community on telegram from 2016 to 2022,
+  promoting the SciPyLA Ambassador program, assisting new chairs in the
+  organization structure of the conferences.
+
+- Regional Support
+
+    The remain funding would be used to maintain the template updated.
+
+- Link to Code of Conduct *: https://opensciencelabs.org/guidelines/mentoring/en/coc/
+
+- Percentage of Python *:
+    100%
+
+- Follow-up activities *:
+
+    As a follow up activity, we plan to have a blog post about all updates done via this grant.
```

Comment:
it is not necessary to add to this grant proposal, but it would be nice to have a blog post for each new feature, so the final blog post would be almost a collection of information of these previous blog posts.